sdk/tables/azure-data-tables/src/main/java/com/azure/data/tables/package-info.java

```diff
@@ -2,6 +2,257 @@
+ *
+ * <hr/>
+ *
+ *
+ * <h3>Create a Table using {@link com.azure.data.tables.TableServiceClient TableServiceClient}</h3>
+ *
+ * <p>The {@link com.azure.data.tables.TableServiceClient#createTable(java.lang.String) TableServiceClient createTable} method can be used to create an new table within your Azure Storage or Azure Cosmos account.</p>
+ *
+ * <p>The following example creates a table with the name "tableName".</p>
+ *
+ * <!-- src_embed com.azure.data.tables.TableServiceClient.createTable.package#String -->
+ * <pre>
+ * tableServiceClient.createTable&#40;&quot;tableName&quot;&#41;;
+ * </pre>
+ * <!-- end com.azure.data.tables.TableServiceClient.createTable.package#String -->
+ *
+ * <p><strong>Note:</strong> For asynchronous sample, refer to {@link com.azure.data.tables.TableServiceAsyncClient TableServiceAsyncClient}</p>
+ *
+ * <hr/>
+ *
+ * <h3>List Tables using {@link com.azure.data.tables.TableServiceClient TableServiceClient}</h3>
+ *
+ * <p>The {@link com.azure.data.tables.TableServiceClient#listTables() TableServiceClient listTables} method can be used to list the tables that are within an Azure Storage or Azure Cosmos account.</p>
+ *
+ * <p>The following example lists all the tables in the account without any filtering of Tables.</p>
+ *
+ * <!-- src_embed com.azure.data.tables.TableServiceClient.listTables.package -->
+ * <pre>
+ * tableServiceClient.listTables&#40;&#41;.forEach&#40;table -&gt; &#123;
+ *     String tableName = table.getName&#40;&#41;;
+ *     System.out.println&#40;&quot;Table name: &quot; + tableName&#41;;
+ * &#125;&#41;;
+ * </pre>
+ * <!-- end com.azure.data.tables.TableServiceClient.listTables.package -->
+ *
+ * <p><strong>Note:</strong> For asynchronous sample, refer to {@link com.azure.data.tables.TableServiceAsyncClient TableServiceAsyncClient}</p>
+ *
+ * <hr/>
+ *
+ * <h3>Delete a Table using {@link com.azure.data.tables.TableServiceClient TableServiceClient}</h3>
+ *
+ * <p>The {@link com.azure.data.tables.TableServiceClient#deleteTable(java.lang.String) TableServiceClient deleteTable} method can be used to delete a table that is within your Azure Storage or Azure Cosmos account.</p>
+ *
+ * <p>The following example deletes a table with the name "tableName".</p>
+ *
+ * <!-- src_embed com.azure.data.tables.TableServiceClient.deleteTable.package#String -->
+ * <pre>
+ * tableServiceClient.deleteTable&#40;&quot;tableName&quot;&#41;;
+ * </pre>
+ * <!-- end com.azure.data.tables.TableServiceClient.deleteTable.package#String -->
+ *
+ * <p><strong>Note:</strong> For asynchronous sample, refer to {@link com.azure.data.tables.TableServiceAsyncClient TableServiceAsyncClient}</p>
+ *
+ * <hr/>
+ *
+ * <h3>Retrieve a {@link com.azure.data.tables.TableClient TableClient} from a {@link com.azure.data.tables.TableServiceClient TableServiceClient}</h3>
```

Comment:
Add an H2 for Table Client Usage/Examples will help to indicate the start of new series of samples here.